src/opnsense/www/js/widgets/IpsecTunnels.js

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2024 Cedrik Pischem
+ * Copyright (C) 2024 Deciso B.V.
```

Comment:
don't think this is needed here now, may need to hook up a proper mail for the first line, but let's discuss first